### PR TITLE
Raising an error if a user tries to load in a Multiple Reservoir model.

### PR DESCRIPTION
**Pull request by cflynn3**

### CODE REVIEW

File: ResSimpy/Nexus/NexusSimulator.py [hunk 1]
```diff
@@ -551,6 +551,8 @@ def __load_fcs_file(self) -> None:
                 value = fo.get_token_value('DEFAULT_UNITS', line, fcs_content_with_includes)
                 if value is not None:
                     self._default_units = UnitSystem(value.upper())
+            elif nfo.check_token(token='RESERVOIR', line=line):
+                raise ValueError('Multiple reservoir models are not currently supported by ResSimpy.')
```
Comment (jonnyh404) [inline ResSimpy/Nexus/NexusSimulator.py:555]: Could be NotImplementedError

**Review by jonnyh404 — APPROVED** [review]
LGMT

File: ResSimpy/Nexus/NexusSimulator.py [hunk 2]
```diff
@@ -551,6 +551,8 @@ def __load_fcs_file(self) -> None:
                 value = fo.get_token_value('DEFAULT_UNITS', line, fcs_content_with_includes)
                 if value is not None:
                     self._default_units = UnitSystem(value.upper())
+            elif nfo.check_token(token='RESERVOIR', line=line):
+                raise NotImplementedError('Multiple reservoir models are not currently supported by ResSimpy.')
```
Comment (kwr3) [inline ResSimpy/Nexus/NexusSimulator.py:555]: good catch for MULTIRES models

**Review by kwr3 — APPROVED** [review]
LGTM

**Pull request merged**